examples/membrane/memv.in

```diff
@@ -9,9 +9,10 @@ glen 300 300 300
 lpbe
 pdie 5
 sdie 80
-bcfl mem
-ion 1.0 0.1 2.0
-ion -1.0 0.1 2.0
+# WARN: bcfl - mem option is no longer supported
+# bcfl mem
```

Comment:
If the `bcfl mem` is no longer supported, then this example is not useful.

Reply:
This was my mistake. `bcfl` is in the spec.